gqueries/general/primary_demand/total_primary_demand_aggregated_carriers/aggregated_total_primary_demand_of_liquid_biofuels.gql

```diff
@@ -0,0 +1,9 @@
+- unit = PJ
+- query = 
+    SUM(
+      Q(total_primary_demand_of_biodiesel),
+      Q(total_primary_demand_of_bio_ethanol),
+      Q(total_primary_demand_of_bio_lng),
```

Comment:
I'm in doubt as to whether bio-LNG should be part of liquid biofuels or of biogas. Technically of course it is a liquid, but LNG we also allocate to natural gas and derivates. Let's discuss.

Reply:
I would say we put it under biogas, since it's main component is [bio-methane](https://nordsol.com/articles/what-is-bio-lng/#:~:text=To%20make%20bio%2DLNG%2C%20the,a%20practically%20carbon%20neutral%20biofuel.)